ci/build_container/build_container.sh

```diff
@@ -43,6 +35,18 @@ make install
 cd ..
 rm -fr libevent*
 
+# boring ssl
+git clone https://boringssl.googlesource.com/boringssl
+cd boringssl
+git reset --hard 78684e5b222645828ca302e56b40b9daff2b2d27
+cmake -DCMAKE_INSTALL_PREFIX:PATH=$THIRDPARTY_BUILD .
```

Comment:
BoringSSL doesn't provide `make install` target, so `-DCMAKE_INSTALL_PREFIX:PATH=$THIRDPARTY_BUILD` is useless and can be removed.